tests/gui/test_widgets.py

```diff
@@ -18,6 +17,7 @@
 
 from securedrop_client import db, logic, storage
 from securedrop_client.app import threads
+from securedrop_client.gui.format_helpers import format_datetime_local
```

Comment:
Per <https://app.circleci.com/pipelines/github/freedomofpress/securedrop-client/3200/workflows/379f7720-ef14-4b46-ac7b-bc95572cff16/jobs/21202/tests>, this should be:

```suggestion
from securedrop_client.gui.datetime_helpers import format_datetime_local
```

Reply:
Oops! Thanks for this, last minute renaming